myLib/dataBase/dbAction.go

```diff
@@ -0,0 +1,76 @@
+package database
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func CheckCreateDB() (*sql.DB, error) {
+
+	// Определение расположения исполняемого файла
+	// Проверка присутствия файла БД
+	appPath, err := os.Executable()
+	if err != nil {
+		return nil, err
+	}
+
+	dbFile := filepath.Join(filepath.Dir(appPath), os.Getenv("DB_NAME"))
+	_, err = os.Stat(dbFile)
+
+	var install bool
+	if err != nil {
+		install = true // Установка признака в необходимости создания БД и её инициализации
+	}
+
+	// Подключение к БД
+	// если БД нет, она создаётся и пингуется
+	db, err := sql.Open(os.Getenv("DB_DRIVER"), os.Getenv("DB_NAME"))
+	if err != nil {
+		return nil, err
+	}
+
+	err = db.Ping()
+	if err != nil {
+		return nil, err
```

Comment:
Лучше все ошибки оборачивать, чтобы быстрее потом понимать где они возникли
```suggestion
		return nil, fmt.Errorf("error in db ping: %w", err)
```